presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergMetadataColumn.java

```diff
@@ -39,7 +40,10 @@ public enum IcebergMetadataColumn
      * Iceberg reserved row ids begin at INTEGER.MAX_VALUE and count down. Starting with MIN_VALUE here to avoid conflicts.
      * Inner type for row is not known until runtime.
      */
-    UPDATE_ROW_DATA(Integer.MIN_VALUE, "$row_id", RowType.anonymous(ImmutableList.of(UNKNOWN)), STRUCT)
+    UPDATE_ROW_DATA(Integer.MIN_VALUE, "$row_id", RowType.anonymous(ImmutableList.of(UNKNOWN)), STRUCT),
+    MERGE_TARGET_ROW_ID_DATA(Integer.MIN_VALUE + 1, "$row_id", RowType.anonymous(ImmutableList.of(UNKNOWN)), STRUCT),
+    MERGE_PARTITION_SPEC_ID(Integer.MIN_VALUE + 2, "partition_spec_id", INTEGER, PRIMITIVE),
+    MERGE_PARTITION_DATA(Integer.MIN_VALUE + 3, "partition_data", VARCHAR, PRIMITIVE)
```

Comment:
```suggestion
    MERGE_PARTITION_SPEC_ID(Integer.MIN_VALUE + 2, "$partition_spec_id", INTEGER, PRIMITIVE),
    MERGE_PARTITION_DATA(Integer.MIN_VALUE + 3, "$partition_data", VARCHAR, PRIMITIVE)
```

nit: following the same naming convention as the other columns?